resources/meetups/2023-02-20-laravel-poland-meetup-24-test.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Why `-test` in filename?

Reply:
Ah, okay. Maybe if it's for testing purposes do not add it to the repository?

Reply:
No problem, I can remove it :)  I wanted you to check how page look when next meetup is set, otherwise everywhere would be empty states